Do not show that airplane mode is enabled on lock screen

The icon can be removed with the "System UI Tuner", so no need to patch this as
well.

Confirmed working on:

* (Android 10, see history of this patch)
* (Android 11, see history of this patch)
* Android 12

project frameworks/base/
diff --git a/frameworks/base/packages/SystemUI/src/com/android/keyguard/CarrierTextManager.java b/frameworks/base/packages/SystemUI/src/com/android/keyguard/CarrierTextManager.java
index cfef6cb399ed..bbb71e079053 100644
--- a/frameworks/base/packages/SystemUI/src/com/android/keyguard/CarrierTextManager.java
+++ b/frameworks/base/packages/SystemUI/src/com/android/keyguard/CarrierTextManager.java
@@ -422,8 +422,7 @@ public class CarrierTextManager {
     }
 
     private String getAirplaneModeMessage() {
-        return mShowAirplaneMode
-                ? getContext().getString(R.string.airplane_mode) : "";
+        return "";
     }
 
     /**
